packages/typespec-go/.scripts/tspcompile.js

```diff
@@ -47,6 +48,7 @@ const httpSpecsGroup = {
   'condreqgroup': ['special-headers/conditional-request'],
   //'repeatabilitygroup': ['special-headers/repeatability'],   // requires union support
   'specialwordsgroup': ['special-words'],
+  'jsonlgroup': ['streaming/jsonl'],
```

Comment:
FYI this produces incorrect codegen and is tracked in https://github.com/Azure/autorest.go/issues/1594